src/eds_files/measuring_distance_node.eds

```diff
@@ -0,0 +1,6 @@
+[2000]
+ParameterName=Distance
+ObjectType=0x7
+DataType=0x0006
```

Comment:
wat voor datatype is "0x0006"? misschien als comment achter zetten? Idem met "ObjectType"

Reply:
ik weet niet zeker of daar ruimte voor is bij een eds file maar zal het bekijken

Reply:
Je kan niet zomaar gaan commenten dat moet in een aparte blok maar ik zal wel informatie toevoegen